integration/tests/conftest.py

```diff
@@ -379,7 +383,11 @@ def account_with_all_tokens(
         100000000,
         web3_client_usdt.chain_id,
     )
-    return neon_account
+
+    yield neon_account
+
+    if environment.use_bank:
+        evm_loader.drain_sol(from_=neon_account, to=bank_account.pubkey())
```

Comment:
neon_account is not solana account 
it doesn't have sols at all

Reply:
we do deposit_wrapped_sol_from_solana_to_neon here , so we need withdraw deposited sol from neon no solana